assets/wizards/connections/views/main/google.js

```diff
@@ -125,6 +134,11 @@ const GoogleOAuth = ( { setError } ) => {
 		}
 		return __( 'Not connected', 'newspack' );
 	};
+
+	if ( ! initialized ) {
+		return null;
```

Comment:
698e8726f7c01810012898fb35e783a31d5a70e0 removes the check. It causes a bit of layout shift to the Ads onboarding in case the OAuth is not available but I don't mind it. WDYT?